### PR TITLE
Issue-1162: Support PHP 8.3

### LINKED ISSUE #1162: Support PHP 8.3
### Description

As titled.

### Use Case

Sites running in PHP 8.3

### PR DESCRIPTION
**Pull request by renatonascalves**
### Summary

Fixes #1162 - Support PHP 8.3

### Description

As titled. The changes include updates to ensure compatibility with PHP 8.3.

### Use Case

Sites running in PHP 8.3

### CODE REVIEW

**Comment by renatonascalves** [comment]
I'll update the branch protections for the new actions once this pr is approved. 🤞🏾

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -16,8 +16,12 @@ jobs:
     strategy:
       matrix:
         multisite: [true, false]
-        php: ['8.2', '8.1', '8.0']
+        php: ['8.3', '8.2', '8.1', '8.0']
         wordpress: ["latest"]
+        include:
+          - php: '8.0'
+            wordpress: '6.3'
+            multisite: false
```
Comment (renatonascalves) [inline .github/workflows/phpunit.yml:24]: This is testing the minimum requirements for Apple News.

File: apple-news.php [hunk 2]
```diff
@@ -51,9 +51,9 @@ function apple_news_date( $format, $timestamp = null, $timezone = null ) {
  */
 function apple_news_activate_wp_plugin() {
 	// Check for PHP version.
-	if ( version_compare( PHP_VERSION, '5.3.6' ) < 0 ) {
+	if ( version_compare( PHP_VERSION, '8.0.0' ) < 0 ) {
 		deactivate_plugins( basename( __FILE__ ) );
-		wp_die( esc_html__( 'This plugin requires at least PHP 5.3.6', 'apple-news' ) );
+		wp_die( esc_html__( 'This plugin requires at least PHP 8.0.0', 'apple-news' ) );
```
Comment (renatonascalves) [inline apple-news.php:56]: I think this is oudated. And it doesn't reflect what we are advertising to support.

File: phpunit.xml.dist [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-	</php>
-	<testsuites>
-		<testsuite name="Unit Tests">
-			<directory prefix="test-" suffix=".php">./tests/</directory>
-		</testsuite>
-	</testsuites>
+  bootstrap="tests/bootstrap.php"
+  backupGlobals="false"
+  colors="true"
+  convertDeprecationsToExceptions="true"
```
Comment (renatonascalves) [inline phpunit.xml.dist:5]: Let's catch deprecations earlier. Currently, none exists.

File: apple-news.php (hunk 2 above)
Comment (kevinfodness) [reply apple-news.php:56]: Good catch!

**Review by kevinfodness — APPROVED** [review]
🍣

**Pull request merged**